src/main/resources/assets/js/app/highchart.es6

```diff
@@ -318,19 +320,42 @@ export function init() {
         const buttonShowGraph = $('button#show-graph-' + highchartsContentKey)
 
         buttonShowDataTable.on('click', (e) => {
-          buttonShowDataTable.addClass('active')
-          buttonShowGraph.removeClass('active')
-          dataTable.show()
-          graph.hide()
+          buttonShowDataTable.addClass('active');
+          buttonShowGraph.removeClass('active');
+          dataTable.removeClass('hide-div');
+          dataTable.addClass('show-div');
+          graph.removeClass('show-div');
+          graph.addClass('hide-div');
         })
 
         buttonShowGraph.on('click', (e) => {
-          buttonShowGraph.addClass('active')
-          buttonShowDataTable.removeClass('active')
-          dataTable.hide()
-          graph.show()
+          buttonShowGraph.addClass('active');
+          buttonShowDataTable.removeClass('active');
+          dataTable.removeClass('show-div');
+          dataTable.addClass('hide-div');
+          graph.addClass('show-div');
+
         })
+
+
+        $(window).resize(function() {
+          $(window).scroll(function (event) {
+            event.isDefaultPrevented();
+            $('html, body').animate({
+              scrollTop: $('#content').offset().top
+            }, 'fast');
+          });
+        });
+
       }
     })
   })
 }
+
+
+
+$( document ).ready(function(event) {
```

Comment:
Denne gir feil i console.log på alle sider. Bør unngå å ha en global funksjon for å løse problemet.